docs/dashboard.md

```diff
@@ -0,0 +1,33 @@
+# SC4S Metrics and Events Dashboard
+SC4S Metrics and Events Dashboard enables monitoring crucial metrics and events flow of all the SC4S instances sending data to a chosen Splunk platform.
```

Comment:
The SC4S Metrics and Events Dashboard lets you monitor crucial metrics and event flows for all the SC4S instances sending data to a chosen Splunk platform.